File: app/src/main/java/com/thatcakeid/zrytezene/UpdateActivity.java
package com.thatcakeid.zrytezene;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class UpdateActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_update);

        ExtraMetadata.setWatermarkColors(findViewById(R.id.text_watermark_update));
    }
}

File: app/src/main/java/com/thatcakeid/zrytezene/UpdateActivity.kt
package com.thatcakeid.zrytezene

import com.thatcakeid.zrytezene.ExtraMetadata.setWatermarkColors
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.thatcakeid.zrytezene.databinding.ActivityUpdateBinding

class UpdateActivity : AppCompatActivity() {
    private val binding by lazy {
        ActivityUpdateBinding.inflate(layoutInflater)
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(binding.root)
        setWatermarkColors(binding.textWatermark, binding.watermarkRoot)
    }
}